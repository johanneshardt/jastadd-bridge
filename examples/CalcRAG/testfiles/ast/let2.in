let r=3.23 p=100.0 in r/p end
