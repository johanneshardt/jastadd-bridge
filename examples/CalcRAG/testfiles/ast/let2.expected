Program
  Let
    List
      Binding
        IdDecl ID="r"
        Numeral NUMERAL="3.23"
      Binding
        IdDecl ID="p"
        Numeral NUMERAL="100.0"
    Div
      IdUse ID="r"
      IdUse ID="p"
